cfpb/consumerfinance.gov
# Contact phones: Fix closing span tag on TTY number

TTY number `<span>` was erroneous a `<span>` instead of `</span>`

## Changes

- Add closing span tag.


## How to test this PR

1. Check a page with TTY phone numbers, like http://localhost:8000/cfpb-ombudsman/ and see that there isn't an empty `<span></span>` at the end of the number
